src/Item.php

```diff
@@ -1199,11 +1217,11 @@ public function get_enclosures()
             // PLAYER
             if ($player_parent = $this->get_item_tags(\SimplePie\SimplePie::NAMESPACE_MEDIARSS, 'player')) {
                 if (isset($player_parent[0]['attribs']['']['url'])) {
-                    $player_parent = $this->sanitize($player_parent[0]['attribs']['']['url'], \SimplePie\SimplePie::CONSTRUCT_IRI);
+                    $player_parent = $this->sanitize($player_parent[0]['attribs']['']['url'], \SimplePie\SimplePie::CONSTRUCT_IRI, $this->get_base($player_parent[0]));
```

Comment:
There were no tests for this section and I have node added any. Help welcome if anyone is motivated.
For sure not providing the base URL will lead to wrong absolute URLs